.project-keeper.yml

```diff
@@ -67,31 +67,74 @@ build:
+              jar_odbc=target/tableau-exasol-connector-odbc-${ver_odbc}.jar
+              tgt_odbc=target/tableau-exasol-connector-odbc-${ver_odbc}.taco
+              cp target/exasol_jdbc.taco ${jar_jdbc}
+              cp target/exasol_odbc.taco ${jar_odbc}
+              echo "jar_jdbc=${jar_jdbc}" >> "$GITHUB_OUTPUT"
+              echo "jar_odbc=${jar_odbc}" >> "$GITHUB_OUTPUT"
+              echo "tgt_jdbc=${tgt_jdbc}" >> "$GITHUB_OUTPUT"
+              echo "tgt_odbc=${tgt_odbc}" >> "$GITHUB_OUTPUT"
+              cat "$GITHUB_OUTPUT"
         - action: INSERT_AFTER
           job: build-and-test
-          stepId: retrieve-code-signing-certificate
+          stepId: esign-filter
           content:
-            name: Retrieve code signing certificate chain
-            id: retrieve-code-signing-certificate-chain
-            run: echo $CODE_SIGNING_CERTIFICATE_CHAIN_BASE64 | base64 --decode > target/cert_chain.p7b
-            env:
-              CODE_SIGNING_CERTIFICATE_CHAIN_BASE64: ${{ secrets.CODE_SIGNING_CERTIFICATE_CHAIN_BASE64 }}
-
+            name: Sign JDBC with CodeSignTool
+            id: esign-jdbc
+            uses: sslcom/esigner-codesign@v1.3.1
+            with:
+              command: sign
+              username: ${{ secrets.ESIGN_USERNAME }}
+              password: ${{ secrets.ESIGN_PASSWORD }}
+              credential_id: ${{ secrets.ESIGN_CREDENTIAL_ID }}
+              totp_secret: ${{ secrets.ESIGN_TOTP_SECRET }}
+              file_path: ${{ steps.esign-filter.outputs.jar_jdbc }}
+              override: true
         - action: INSERT_AFTER
           job: build-and-test
-          stepId: retrieve-code-signing-certificate-chain
+          stepId: esign-jdbc
           content:
-            name: Sign connectors
-            id: sign-connectors
-            run: ./tools/sign_connector.sh target/cert.p12 target/cert_chain.p7b
-            env:
-              CODE_SIGNING_CERTIFICATE_PASSWORD: ${{ secrets.CODE_SIGNING_CERTIFICATE_PASSWORD }}
+            name: Sign ODBC with CodeSignTool
+            id: esign-odbc
+            uses: sslcom/esigner-codesign@v1.3.1
+            with:
+              command: sign
+              username: ${{ secrets.ESIGN_USERNAME }}
+              password: ${{ secrets.ESIGN_PASSWORD }}
+              credential_id: ${{ secrets.ESIGN_CREDENTIAL_ID }}
+              totp_secret: ${{ secrets.ESIGN_TOTP_SECRET }}
+              file_path: ${{ steps.esign-filter.outputs.jar_odbc }}
+              override: true
+        - action: INSERT_AFTER
+          job: build-and-test
+          stepId: esign-odbc
+          content:
+            name: Copy signed files
+            id: esign-copy-target
+            run: |
+              cp ${{ steps.esign-filter.outputs.jar_jdbc }} ${{ steps.esign-filter.outputs.tgt_jdbc }}
+              cp ${{ steps.esign-filter.outputs.jar_odbc }} ${{ steps.esign-filter.outputs.tgt_odbc }}
+              echo "All done"
+              ls -l target
```

Comment:
Add step to verify the signature, see https://github.com/exasol/tableau-connector/blob/main/tools/sign_connector.sh#L46-L58